src/css.js

```diff
@@ -58,12 +56,11 @@ function vendorPropName( name ) {
 	}
 }
 
-function setPositiveNumber( elem, value, subtract ) {
-	var matches = rnumsplit.exec( value );
```

Comment:
Stripping this `exec` will break our [other application](https://github.com/timmywil/jquery/blob/width-1712/src/css.js#L394) of `setPositiveNumber`.